easybuild/easyconfigs/c/CrayPGI/CrayPGI-18.07.eb

```diff
@@ -15,7 +15,7 @@ dependencies = [
     ('PrgEnv-pgi', EXTERNAL_MODULE),
     ('atp/2.1.2', EXTERNAL_MODULE),
     ('pgi/18.3.0', EXTERNAL_MODULE),
-    ('cray-libsci/18.07.1', EXTERNAL_MODULE),
+#    ('cray-libsci/18.07.1', EXTERNAL_MODULE),
```

Comment:
Can you please delete this line instead of commenting it out?

Reply:
I would rather keep it as a reminder that we should not have libsci. 
One recently felt in that trap.

Reply:
I suggest to explain in a comment why the line is commented out

Reply:
Since libsci is not part of the default PrgEnv-pgi, please just remove it. 

The only reason why one felt in the trap, is because the was a commented out entry without any explanation.

If you wish to add reminders on the files they should be understandable to everyone and not only for you.

We always create new recipes based on the latest available, if libsci is not listed originally then we're safe.